proposals/type-classes.md

```diff
@@ -256,9 +256,8 @@ The above instances are each defined alongside their respective type definitions
 
 To determine whether a type class definition is a valid type-side implementation we perform the following check:
 
-
-1. A "local type" is any type (but not typealias) defined in the current file (e.g. everything defined in `data.collections.bar` if we're evaluating `data.collections.bar`).
-2. A generic type parameter is "covered" by a type if it occurs within that type, e.g. `MyType` covers `T` in `MyType<T>` but not `Pair<T, MyType>`.
+1. A "local type" is any type (but not type alias) defined in the current file (e.g. everything defined in `data.collections.bar` if we're evaluating `data.collections.bar`).
+2. A generic type parameter is "covered" by a type if it occurs within that type (e.g. `MyType` covers `T` in `MyType<T>` but not `Pair<T, MyType>`).
```

Comment:
Added the example back within parentheses (as I did in my original commit) - this keeps with how it's done in the line above.